build_config.py

```diff
@@ -6,9 +6,26 @@
 build_base_dir = 'bin'
 target_name = 'mip_upe_sdk'
 
-# Scons does not have built-in support to create multiple object files at the same time.
-# This helper fills that gap.
+INCLUDE_PATH_BASE = [
+    '#sdk',
+    '#googletest',
+    '#googletest/include',
+    '#third_party',
+    '#third_party/include',
+]
+
+LIBXML2FLAVOR = "libxml2-2.9.3-win32-x86_64"
+LIBXML2HEADERPATH = "#src/external/libxml2/" + LIBXML2FLAVOR + "/include/libxml2"
```

Comment:
Why are we having this in the build config and not where it is needed?

Reply:
it is added in the SConstruct and all the files use these paths, so I added them here. More specific paths are added where they are needed